src/components/new-safe/steps/Step2/index.tsx

```diff
@@ -42,13 +48,9 @@ const CreateSafeStep2 = ({ onSubmit, onBack, data }: StepRenderProps<NewSafeForm
 
   const { fields: ownerFields, append: appendOwner, remove: removeOwner } = useFieldArray({ control, name: 'owners' })
 
-  const {
-    fields: mobileOwnerFields,
-    append: appendMobileOwner,
-    remove: removeMobileOwner,
-  } = useFieldArray({ control, name: 'mobileOwners' })
+  const allOwners = [...ownerFields]
```

Comment:
You don't need this anymore as you don't need to combine the owner(s) and mobile owner(s)